list/list.js

```diff
@@ -273,6 +275,106 @@ var DefineList = Construct.extend("DefineList",
 			}
 			return this;
 		},
+		/**
+		 * @function can-define/list/list.prototype.assign assign
+		 * @parent can-define/list/list.prototype
+		 *
+		 * Sets an item or property or items or properties on a list.
+		 *
+		 * @signature `list.assign(newProps)`
+		 *
+		 * Updates the properties on the list with `newProps`.
+		 *
+		 * ```js
+		 * var list = new DefineList(["A","B"]);
+		 * list.assign({count: 1000, skip: 2});
+		 * list.get("count") //-> 1000
+		 * ```
+		 */
+		assign: function(prop) {
+			if (canReflect.isListLike(prop)) {
+				canReflect.assignList(this, prop);
+			} else {
+				canReflect.assignMap(this, prop);
+			}
+			return this;
+		},
+		/**
+		 * @function can-define/list/list.prototype.assign update
+		 * @parent can-define/list/list.prototype
+		 *
+		 * Sets an item or property or items or properties on a list.
+		 *
+		 * @signature `list.update(newProps)`
+		 *
+		 * Updates the properties on the list with `newProps`. Properties not in `newProps` will be set to `undefined`.
+		 *
+		 * ```js
+		 * var list = new DefineList(["A","B"]);
+		 * list.assign({count: 0, skip: 2});
+		 * list.update({count: 1000});
+		 * list.get("count") //-> 1000
+		 * list.get("skip") //-> undefined
+		 * ```
+		 */
+		update: function(prop) {
+			if (canReflect.isListLike(prop)) {
+				canReflect.updateList(this, prop);
+			} else {
+				canReflect.updateMap(this, prop);
+			}
+			return this;
+		},
+		/**
+		 * @function can-define/list/list.prototype.assign assignDeep
+		 * @parent can-define/list/list.prototype
+		 *
+		 * Sets an item or property or items or properties on a list.
+		 *
+		 * @signature `list.assignDeep(newProps)`
+		 *
+		 * Updates the properties on the list with `newProps`.
+		 *
+		 * ```js
+		 * var list = new DefineList(["A","B"]);
+		 * list.assignDeep({count: 1000, skip: 2});
```

Comment:
Can we make this example or description more clearly explain the difference between this and `assign`? Right now the example is identical.